test/integration/analytics_int_test.go

```diff
@@ -571,7 +571,7 @@ func (suite *AnalyticsIntegrationTestSuite) TestHeapEvents() {
 	)
 
 	cp.Expect("Creating a Virtual Environment")
-	cp.Expect("Activated", e2e.RuntimeSourcingTimeoutOpt)
```

Comment:
Is this really okay? We're doing an activate, so it will take time to source the runtime before printing "Activated".

Reply:
Whoops, this might have been a mistake.

Reply:
It turns out the diff isn't showing the lines above. We aren't actually sourcing a runtime in this test so I think this is appropriate.